spec/requests/tenant_quotas_spec.rb

```diff
@@ -131,6 +131,12 @@
       expect(response).to have_http_status(:no_content)
     end
 
+    it "supports OPTIONS requests on a subcollection without authorization" do
+      options api_tenant_quotas_url(:c_id => tenant.id)
```

Comment:
I thought this was supposed to be api_tenant_quotas_url(nil, tenant)

look at other uses, like api_provider_flavors_url(nil, ems).